Form-Controls/index.html

```diff
@@ -1,28 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Luke form exercise</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <div class="form_container">
+        <form>
+            <div class="name">
+            <label for="name">Name: </label>
+            <input id="name" placeholder="Your name" type="text" required>
+          </div>
+          <br />
+          <div class="email">
+            <label for="email">Email: </label>
+            <input id="email" type="email" placeholder="Your email" required>
+          </div>
+          <div class="color">
+            <p>Please select your favorite color.</p>
+            <input type="radio" id="red" name="color" value="red">
+            <label for="red">Red</label><br />
+            <input type="radio" id="green" name="color" value="green">
+            <label for="green">Green</label><br />
+            <input type="radio" id="blue" name="color" value="blue">
+            <label for="blue">Blue</label><br />
+          </div>
+          <br />
+          <div class="size">
+            <label for="size_choice">Choose a size.</label>
+            <select name="size" id="size" required>
```

Comment:
Touch targets do not have sufficient size or spacing. This is having an impact on your accessibility score.

Reply:
Thank you @Waldo-Strydom, this has been fixed.